src/xeus_sqlite_interpreter.cpp

```diff
@@ -658,7 +663,7 @@ namespace xeus_sqlite
         banner.append(SQLite::VERSION);
 
         result["banner"] = banner;
-        result["language_info"]["name"] = "sqlite3";
+        result["language_info"]["name"] = "text";
```

Comment:
Could we use some other `sql` flavor there instead?

Reply:
Just pushed it :)
It works well!